.github/workflows/ci.yaml

```diff
@@ -35,14 +35,13 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ['8.0', '8.1']
+                php: ['8.0', '8.1', '8.2', '8.3']
         steps:
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
                   php-version: ${{ matrix.php }}
                   extensions: json
-                  tools: prestissimo
```

Comment:
not longer required since composer 2: https://github.com/hirak/prestissimo#announcement-composer-2-is-now-available